---
layout: page
title: Dingfeng Wu Lab
subtitle: Bioinformatics, Metagenomics and Artificial intelligence
cover-img: /assets/img/hanzhou.png
full-width: true
language: utf8
---

<div class="container-lg">
    <div class="row" style="margin-top: 0rem">
        <h2>Research focus</h2>
        <p class="text-justify">The Wu Lab focuses on the integration and analysis of omics data (genome, transcriptome, proteome, metabolome, macrogenome, etc.), especially the research of metagenomics.
            The Wu Lab uses machine learning, deep learning, artificial intelligence and causal inference algorithms for development and application.
            And we are also engaged in computer-aided drug design, including TCM informatics and TCM modernization research.
        </p>
    </div>
</div>

<div class="container-lg">
    <div class="row"  style="margin-top: 3rem">
        <div class="col-lg-4">
            <div class="text-center"><img alt="Human microbiome" class="rounded-circle shadow" src="/assets/img/bioinformatics2.png" width="200px"></div>
            <h2 class="text-center">Bioinformatics</h2>
            <p class="text-justify" style="height:250px; overflow: hidden;">Bioinformatics plays a crucial role in the understanding of genetics and molecular biology and is used to develop new techniques for drug discovery, gene therapy, and personalized medicine. We use bioinformatics technology to integrate omics data, research on disease mechanism, computer-aided drug design and other research.</p>
            <p class="text-center" style="margin-bottom: 1rem"> <a class="btn btn-primary" href="#" role="button">More about our research »</a> </p>
        </div>
        <div class="col-lg-4">
            <div class="text-center"><img alt="Bioinformatics" class="rounded-circle shadow" src="/assets/img/metagenomics2.png" width="200px"></div>
            <h2 class="text-center">Metagenomics</h2>
            <p class="text-justify" style="height:250px; overflow: hidden;">Metagenomics provides a comprehensive view of the diversity of microorganisms in a given environment. We focuse on studying the mechanism of gut microbiome in nonalcoholic fatty liver disease, inflammatory bowel disease, coronary heart disease, tumor and other diseases, exploring biomarker and providing new treatment strategies.</p> 
            <p class="text-center" style="margin-bottom: 1rem"> <a class="btn btn-primary" href="#" role="button">More about our research »</a> </p>
        </div> 
        <div class="col-lg-4"> 
            <div class="text-center"><img alt="Ocean microbiome" class="rounded-circle shadow" src="/assets/img/artificial_intelligence.png" width="200px"></div>
            <h2 class="text-center">Artificial Intelligence</h2> 
            <p class="text-justify" style="height:250px; overflow: hidden;">Artificial Intelligence (AI) was used in the field of biology to solve complex biological problems. It can analyze of vast amounts of data, make predictions and decisions, and provide new insights into complex biological systems. We use AI technology to develop new algorithms for drug discovery, predictive medicine and microbial ecology.</p> 
            <p class="text-center" style="margin-bottom: 1rem"> <a class="btn btn-primary" href="#" role="button">More about our research »</a> </p>
        </div> 
    </div>
</div>

<div class="container-lg">
    <hr/>
    <div class="row">
        <h3 class="text-justify">Targeting keystone species helps restore the dysbiosis of butyrate-producing bacteria in nonalcoholic fatty liver disease</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 3.5rem;" src="/assets/img/research/research_nafld_keystone.png">
            </div>
            <div class="col-lg-8">
                <p class="text-justify" style="margin-top: 1.5rem;">The dysbiosis of the gut microbiome is one of the pathogenic factors of nonalcoholic fatty liver disease (NAFLD) and also affects the treatment and intervention of NAFLD. Among gut microbiomes, keystone species that regulate the integrity and stability of an ecological community have become the potential intervention targets for NAFLD. An algorithm was implemented to identify keystone species based on causal inference theories and dynamic intervention simulation. Our findings demonstrate the importance of keystone species in restoring the microbial composition toward a normal gut microbiome, suggesting a novel potential microbial treatment for NAFLD.
                </p>
            </div>
        </div>
    </div>
    <div class="row">
        <h3 class="text-justify">Dr AFC: drug repositioning through anti-fibrosis characteristic</h3>
        <div class="row">
            <div class="col-lg-8">
                <p class="text-justify" style="margin-top: 1.5rem;">In this study, anti-fibrosis knowledge base was first built by collecting data from multiple resources. Both structural and biological profiles were then derived from the knowledge base and used for constructing machine learning models including Structural Profile Prediction Model (SPPM) and Biological Profile Prediction Model (BPPM). Three external public data sets were employed for validation purpose and further exploration of potential repositioning drugs in wider chemical space. The resulting SPPM and BPPM models achieve area under the receiver operating characteristic curve (area under the curve) of 0.879 and 0.972 in the training set, and 0.814 and 0.874 in the testing set. Web server: <a href="https://www.biosino.org/drafc">https://www.biosino.org/drafc</a>.
                </p>
            </div>
            <div class="col-lg-4">
                <img style="margin-top: 3rem;" src="/assets/img/research/research_drafc.png">
            </div>
        </div>
    </div>
    <div class="row">
        <h3 class="text-justify">Phylogenetic analyses with systematic taxon sampling show that mitochondria branch within Alphaproteobacteria</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 3rem;" src="/assets/img/research/research_NEE.png">
            </div>
            <div class="col-lg-8">
                <p class="text-justify" style="margin-top: 1.5rem;">Though it is well accepted that mitochondria originated from an alphaproteobacteria-like ancestor, the phylogenetic relation- ship of the mitochondrial endosymbiont to extant Alphaproteobacteria is yet unresolved. The focus of much debate is whether the affinity between mitochondria and fast-evolving alphaproteobacterial lineages reflects true homology or artefacts. We demonstrate that site-exclusion methods produce erratic phylogenetic estimates of mitochondrial origin. Thus, previous phylogenetic hypotheses on the origin of mitochondria based on pretreated datasets should be re-evaluated. We applied alternative strategies to reduce phylogenetic noise by systematic taxon sampling while keeping site substitution information intact. Cross-validation based on a series of trees placed mitochondria robustly within Alphaproteobacteria, sharing an ancient common ancestor with Rickettsiales and currently unclassified marine lineages. 
                </p>
            </div>
        </div>
    </div>
</div>
